src/openMVG/types.hpp

```diff
@@ -17,7 +17,7 @@
 #include <set>
 #include <vector>
 
-#if defined OPENMVG_STD_UNORDERED_MAP
+#ifdef OPENMVG_STD_UNORDERED_MAP
```

Comment:
Yes but `OPENMVG_STD_UNORDERED_MAP` is not currently defined anywhere, I think it comes from the time c++11 was not mandatory. In that case it can be even removed and `unordered_map`used by default. Unless it is left as test code. In that case I can add it as an option I guess